Cargo.toml

```diff
@@ -16,19 +16,18 @@
 # under the License.
 
 [package]
-authors = ["Apache Software Foundation <dev@opendal.apache.org>"]
+authors = ["OpenDAL Contributors <dev@opendal.apache.org>"]
```

Comment:
We don't use ASF because an incubating project is not fully endorsed by the ASF and I'd suggest we identify our name. "Apache OpenDAL (Incubating) Contributors" is wordy. Maybe we can add "Apache" after graduated.